test/python/unittest/API/APITest.py

```diff
@@ -146,25 +146,36 @@ def testData(self):
         # Get a particle species.
         electrons = i.particles["electrons"]
         self.assertIsInstance(electrons, openPMD.ParticleSpecies)
+        pos_x = electrons["momentum"]["x"]
+        w = electrons["weighting"][openPMD.Record_Component.SCALAR]
```

Comment:
I wonder if we could realize this as well was `w = electrons["weighting"]` somehow or expose the constant magic attribute directly in `openPMD.SCALAR`

Reply:
```w = electrons["weighting"]``` will very likely not be possible. On first access, there is ambiguity whether you want to create a ```Record``` or a scalar ```RecordComponent```.
While it would be reasonable to assume a scalar and then later discard that scalar if a named ```RecordComponent``` is added, this will result in runtime failures (not all backends support node deletion). *Note that the opposite is currently assumed.*

I'd rather keep a constant. [Explicit is better than implicit.](https://www.python.org/dev/peps/pep-0020/)

Reply:
ok, then let's probably just define a shortcut as mentioned above